test/chi/components/header.pug

```diff
@@ -3,561 +3,491 @@ title: Header
-        span.chi-header__title.-text--muted Application Title
+    nav#drawer-trigger-3.chi-header__toolbar.-d--flex.-d-lg--none.chi-drawer__trigger(data-target='#drawer-3')
+      .chi-header__start
+        i.chi-icon.icon-map-marker.-mx--1.-ml-lg--0
+        span Menu
       .chi-header__end
-        a.chi-button.-flat(href="#") Login
-        a.chi-button.-brand.-ml--1(href="#") Sign Up
+        i.chi-icon.icon-chevron-down.-mx--1.-mr-lg--0
+  // Drawer
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      #drawer-1.chi-drawer.-left.-menu.-position--absolute
+        .chi-drawer__header
+          button.chi-button.-icon.-close(aria-label='Close')
+            .chi-button__content
+              i.chi-icon.icon-x
+        .chi-drawer__content
+          .-px--2.-text Drawer content here
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      #drawer-0.chi-drawer.-right.-menu.-position--absolute
+        .chi-drawer__header
+          button.chi-button.-icon.-close(aria-label='Close')
+            .chi-button__content
+              i.chi-icon.icon-x
+        .chi-drawer__content
+          .-px--2.-text Drawer content here
+  .chi-backdrop.-closed
+    .chi-backdrop__wrapper
+      #drawer-3.chi-drawer.-bottom.-menu.-position--absolute
+        button.chi-button.-icon.-close(aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+        .chi-drawer__content
+          .-px--4.-py--2
+            ul.chi-tabs.-vertical.-lg
+              li.-active
+                a(href='#') Item
+              li
+                a(href='#') Item
+              li
+                a(href='#') Item
+              li
+                a(href='#') Item
 
 h3 Inverse
-.test-header-base.-w--100
-  header.chi-header.-inverse
-    nav.chi-header__content
+header.chi-header.-inverse
```

Comment:
the parent div is missing and the previous classname was missing -inverse. we'll need to add `.test-header-base-inverse.-w--100` back in order to properly test this component.